reana/reana_dev/wiki.py

```diff
@@ -0,0 +1,148 @@
+            "packages": {
+                "reana-commons": {},
+                "reana-db": {},
+                "reana-job-controller": {},
+                "reana-message-broker": {},
+                "reana-server": {},
+                "reana-ui": {},
+                "reana-workflow-controller": {},
+                "reana-workflow-engine-cwl": {},
+                "reana-workflow-engine-serial": {},
+                "reana-workflow-engine-yadage": {},
+                "reana-workflow-monitor": {},
+            },
+        },
+        "developers": {
+            "title": "For developers",
+            "description": "Understand REANA source code, adapt it to your needs, contribute changes back.",
+            "packages": {"reana": {}, "pytest-reana": {}},
+        },
+        "environments": {
+            "simple": True,
+            "title": "Environments",
+            "description": "Selected containerised environments.",
+            "packages": {
+                "reana-env-aliphysics": {},
+                "reana-env-jupyter": {},
+                "reana-env-root6": {},
+                "reana-auth-krb5": {},
+                "reana-auth-vomsproxy": {},
+            },
+        },
+        "examples": {
+            "simple": True,
+            "title": "Examples",
+            "description": "Selected reusable analysis examples.",
+            "packages": {
+                "reana-demo-alice-lego-train-test-run": {},
+                "reana-demo-alice-pt-analysis": {},
+                "reana-demo-atlas-recast": {},
+                "reana-demo-bsm-search": {},
+                "reana-demo-cdci-crab-pulsar-integral-verification": {},
+                "reana-demo-cdci-integral-data-reduction": {},
+                "reana-demo-cms-h4l": {},
+                "reana-demo-cms-reco": {},
+                "reana-demo-cms-dimuon-mass-spectrum": {},
+                "reana-demo-fcchh-fullsim": {},
+                "reana-demo-helloworld": {},
+                "reana-demo-lhcb-d2pimumu": {},
+                "reana-demo-lhcb-mc-production": {},
+                "reana-demo-root6-roofit": {},
+                "reana-demo-worldpopulation": {},
+            },
+        },
+    }
+
+    def _print_section(data):
+        click.echo(f"### {data['title']}\n")
+        click.echo(f"{data['description']}\n")
+        _print_table(data["packages"], simple=data.get("simple"))
+        click.echo()
+
+    def _print_header(hs):
+        header = separator = "|"
+        for h in hs:
+            header += f" {h} |"
+            separator += f" {'-' * len(h)} |"
+        click.echo(header)
+        click.echo(separator)
+
+    def _print_table(components, simple=False):
+        if simple:
+            headers = ["Package", "Build", "Version"]
+        else:
+            headers = [
+                "Package",
+                "`maint-0.7`",
```

Comment:
What do you think about extracting `maint-0.7` to a variable? e.g. `MAINT_BRANCHES = ["maint-0.7"]` so when:
- A new maint branch is created we just add it to the list.
- When we drop support for a given version, we just remove it from the list.

(_affecting code below_)